readme.txt

```diff
@@ -105,105 +105,87 @@ There are opportunities for developers at all levels to contribute. [Learn more
 
 = 4.7 =
 
-* Release date:
-* Release post:
+**Performance Improvements**
 
-**New Features and Improvements**
+* Experience better performance with single page load caching of the media summary.
+* We made some improvements to avoid slow queries on sites with thousands of images.
+* The Top Posts widget now utilizes an endpoint from the WP.com REST API improving performance, and update flexibility.

```

Comment:
The wording for `improving performance, and update flexibility` seems a bit confusing for me. Could we maybe remove `and update flexibility`?